frontend/src/hooks/usePersonnel.tsx

```diff
@@ -28,12 +29,17 @@ const usePersonnel = (): {
         const response =
           program && (await AxiosPrivate.get(`/${program}/${profileId}`));
         response && setPersonnel(response.data);
+        setRefetch(false);
```

Comment:
If we set refetch to false, wouldn't this trigger another fetch? And then another, and then....

Reply:
Oh I guess not, since fetch is already false. Since fetch is false, I am not sure i see the purpose in setting it to false here?

Reply:
Okay I see why - because we're calling setRefetch(true) below. 

I would feel better about removing the setRefetch(false) and instead in the fetch() function calling setRefetch(!refetch). Reason being, we're calling this elsewhere, so it is entirely possible that the useEffect will be called when setRefetch is true, and would then trigger re-render and re-fetch. I just don't think we can rely on refetch always being false here?

Reply:
That's smarter actually yeah. I'll add that!